docs/index.html

```diff
@@ -523,10 +587,31 @@
       {
         quadrant: 0,
         ring: 2,
-        label: "TypeScript",
+        label: "Kotlin Script",
+        active: true,
+        moved: 0
+      },
+      {
+        quadrant: 0,
+        ring: 2,
+        label: "Kotlin Multiplatform",
```

Comment:
is there some (wiki) page with information for what use-cases this is in "trial"

Reply:
It is in "assess". We got the input from dah that they want to have a look at it

Reply:
ah mixed up the rings. So just some early look at it.